docs/src/pages/guides/flow.md

```diff
@@ -3,9 +3,11 @@
 You can add static typing to JavaScript to improve developer productivity and code quality thanks to [Flow](https://github.com/facebook/flow).
 Have a look at the [Create React App with Flow](https://github.com/mui-org/material-ui/tree/v1-beta/examples/create-react-app-with-flow) example.
 
-## Warning 
-An [existing bug in flow regarding the use of higher order components (HOC)](https://github.com/facebook/flow/issues/5382) 
-limits the usefulness of flow by an application.  As of November 27, 2017 we cannot recommend the use of flow 
-typing in your application in conjunction with `material-ui`.  
+## Warning
+
+An [existing bug in flow regarding the use of higher order components (HOC)](https://github.com/facebook/flow/issues/5382)
+limits the usefulness of flow by an application.
+As of November 27, 2017 we cannot recommend the use of flow typing in your application in conjunction with `material-ui`.
+We encourage people having a look at [TypeScript](/guides/typescript).
```

Comment:
@rosskevin 🗡 👼

Reply:
"We encourage you to have a look at"

Reply:
"limits the usefulness of flow to an application"